src/cli.c

```diff
@@ -1829,6 +1864,103 @@ get_conf_path_result(struct json* j, uintptr_t* r)
    return 1;
 
 }
+static int
+process_alias_result(SSL* ssl, int socket, int32_t output_format)
+{
+   struct json* json = NULL;
+   int result = 0;
+
+   if (pgagroal_management_read_json(ssl, socket, NULL, NULL, &json))
+   {
+      result = 1;
+      goto cleanup;
+   }
+
+   if (output_format == MANAGEMENT_OUTPUT_FORMAT_JSON)
+   {
+      pgagroal_json_print(json, FORMAT_JSON);
+      goto cleanup;
+   }
+
+   // Text format output
+   struct json* output = (struct json*)pgagroal_json_get(json, "output");
```

Comment:
Define variable at the start of the function such that all can be cleaned up in `error:`